Add key definitions for seats and collections (#800)
directus/docs: content/licensing/1.overview.md

Quick links:

- [What counts as a user seat or a collection?](#key-definitions)
- [Enquire about a commercial license](https://directus.com/contact)
- [Upgrading to Directus 12 (existing customer)](#upgrading-to-directus-12)

## Key Definitions

Plan limits are counted in units that are easy to misread. This section defines what counts toward **user seats** and **collections**. For the limits that apply to each tier, see the [pricing page](https://directus.com/pricing).

### User Seats

> A seat is consumed by each **active Studio user**: a user who can access the Data Studio. A user is a Studio user if they have at least one policy with **App Access** or **Admin Access** enabled. See [Studio Users](/guides/auth/access-control#studio-users) for the full criteria.

Reviewer: Don't believe this provides any meaningful info
```suggestion
A seat is consumed by each **active Studio user**. A user is a Studio user if they have at least one policy with **App Access** or **Admin Access** enabled. See [Studio Users](/guides/auth/access-control#studio-users) for the full criteria.
```

**API-only users do not count toward your seat limit.** These are users created and managed in Directus who never access the Data Studio and reach your data only through the REST, GraphQL, WebSockets, or SDK APIs. Common examples:

- End users of your application, whether they register themselves through your frontend or you create their accounts for them.
- Service accounts and integrations that authenticate with a static token.

> None of these consume a seat, however many of them you have, provided no policy grants them app access. Conversely, a user whose policy grants app access counts whether or not they ever log in.

Reviewer: ```suggestion
These users do not consume a seat, regardless of how many exist, provided no policy grants them app access. Conversely, any user whose policy grants app access consumes a seat, whether or not they ever log in.
```

::callout{icon="i-lucide-info"}
**Seats measure Studio access, not the size of your user base.**
<br>

> A project with 2 content editors and 50,000 registered application users consumes 2 seats. Only accounts that can open the Data Studio are counted.

Reviewer: ```suggestion
A project with 2 content editors and 50,000 registered application users consumes 2 seats. Only accounts that can access the Data Studio are counted.
```

::

> Invitations are not blocked when you are at your seat limit, but **activation** of a user beyond the limit is. To activate a pending user, either increase your seat count or free a seat by removing app access from an existing user.

Reviewer: ```suggestion
You can send invitations even after reaching your seat limit. However, you cannot **activate** a user beyond the limit. To activate a pending user, either increase your seat count or free a seat by removing app access from an existing user.
```

### Collections

A collection counts toward your limit if it is a user collection registered in Directus, meaning it has Directus metadata and appears in the Data Studio. See [Collections](/guides/data-model/collections) for the difference between user and system collections.

Counted:

- User collections, whether created through the Data Studio, the API, or created in your database and then registered in Directus.
- Translation collections, such as `articles_translations`.
- Junction tables created for many-to-many and many-to-any relationships.

Not counted:

- System collections, which are prefixed with `directus_`.
- Folders, which group collections in the Data Model settings and are not tables.

> - Database-only tables, meaning tables that exist in your database but have no Directus metadata because they were never registered in Directus.

Reviewer: ```suggestion
- Database-only tables, which exist in your database but have no associated Directus metadata because they are not registered in Directus.
```
